main/glossary/README.md

```diff
@@ -67,151 +102,362 @@ synchronously compared for structural equivalence.
+
+An `invitation`'s properties are:
+- The contract's installation in Zoe, including access to its source code.
+- The contract instance this invitation is for.
+- A handle used to refer to this invitation.
+- A description of this invitation's purpose.
+
+## InvitationIssuer
+
+Since [invitations](#invitation) are special cases of [payments](#payment), invitations 
+must have a dedicated [issuer](#issuer), which is the InvitationIssuer.
+
+Zoe has a single `InvitationIssuer` for its entire lifetime. By having a reference to Zoe, 
+a user can get the `InvitationIssuer`. This lets them claim any invitation they 
+receive from someone else by calling `E(invitationIssuer).claim()` with the untrusted
+invitation as the argument. During the claiming process, the invitationIssuer validates
+the invitation. A successful claim also means that invitation is exclusively yours.
 
 ## Issuer
-Issuers are linked to a single [mint](#mint) and vice versa, so each issuer works
+Issuers are a one-to-one relationship with both a [mint](#mint) and a [brand](#brand), so each issuer works
 with one and only one asset type, such as only working with quatloos or only working
-with moola. They can create empty [purses](#purse) and [payments](#payment) for
-their asset type, but cannot mint new [amounts](#amounts). 
+with moola. This association cannot change to another type. 
 
-Issuers can also transform
+Issuers can create empty [purses](#purse) and [payments](#payment) for
+their asset type, but cannot mint new [amounts](#amount). Issuers can also transform
 payments of their asset type (splitting, combining, burning, and exclusively claiming
 payments). An issuer from a trusted source can determine if an untrusted payment of
 its asset type is valid. 
 
-See the [ERTP Guide's Issuer section](/ertp/guide/issuers-and-mints.md)
-and the [ERTP API's Issuer section](/ertp/api/issuer.md) for more information.
+For more information, see the [ERTP Guide's Issuer section](/ertp/guide/issuers-and-mints.md)
+and the [ERTP API's Issuer section](/ertp/api/issuer.md).
+
+## Keywords
+
+Keywords are unique identifiers per contract. They tie together the [proposal](#proposal), [payments](#payment) 
+to be [escrowed](#escrow), and [payouts](#payout) to the user by serving as keys for key-value pairs in various 
+records with values of [amounts](#amount), [issuers](#issuer), etc.
 
 ## Mint
-In ERTP, mints create digital assets and are the only objects with the authority to do so. 
-Access to a mint gives you the power to create more digital assets of its type at will. Mints
-can only create one type of asset. 
+Agoric has two mint objects, *ERTP mints* and *Zoe Contract Facet mints (ZCFMints)*. They both create
+digital assets. Which mint type creates an asset doesn't matter; quatloos created by an ERTP mint are 
+indistinguishable from quatloos created by a ZCFMint. 
 
```

Comment:
I think this might cause more confusion than it clears up. Let's say that there is only one kind of mint, the ERTP mint, but ZCF provides a different interface, the ZCFMint interface, with an ERTP mint underneath. Both create ERTP assets.

Reply:
How's this: 
[ERTP](#ertp) has a *mint* object, which creates digital assets. [ZCF](#zcf) provides a 
different interface to an ERTP mint, called a *ZCFMint*. It doesn't matter whether you
use the ERTP mint interface (henceforth called an *ERTP mint*) or the ZCFMint interface;
quatloos created by an ERTP mint are indistinguishable from quatloos created by a ZCFMint.

Reply:
I wouldn't put it that way, since any particular mint is going to be of one kind or another, and there won't be duplicate mints for any brand.

assets, brands, and issuers associated with a mint (retrieved with `getIssuerRecord()`) created within a contract are indistinguishable from those created using makeIssuerKit.